commodore/catalog.py

```diff
@@ -124,3 +128,27 @@ def catalog_list(cfg):
             click.echo(f" - {display_name}")
         else:
             click.echo(catalog_id)
+
+
+def catalog_save(cfg, output):
+    clusters = lieutenant_query(cfg.api_url, cfg.api_token, "clusters", "")
+    tenant_ids = {}
+    for cluster in clusters:
+        if "tenant" not in cluster:
+            raise click.ClickException(f"cluster {cluster} does not have a tenant reference")
+        tenant_ids[cluster["tenant"]] = True
+        cluster["gitRepo"]["deployKey"] = "<hidden>"
+        cluster["gitRepo"]["hostKeys"] = "<hidden>"
+    tenants = []
+    for tenant_id in tenant_ids:
+        tenant_response = lieutenant_query(
+            cfg.api_url, cfg.api_token, "tenants", tenant_id
+        )
+        tenants.append(tenant_response)
+    cluster_config = {
+            "clusters": clusters,
+            "tenants": tenants,
+            }
```

Comment:
This would be different from what lieutenant returns. The idea is to have the local case behave like it would talk to Lieutenant. In a future version, we could even make a plugin system to get the configuration from somewhere else (Lieutenant and LocalFile would just be two plugins).
In the rest of the code, I expect the local file to behave the same as Lieutenant.
If we change the structure of this file, we would need to revert it in the code to match the response from Lieutenant. This would actually make the code more complex, without much benefit for commodore.